optimizers/prioritized.js

```diff
@@ -0,0 +1,363 @@
+var path = require("path");
```

Comment:
Then we can separate tests of the actual bundle function from the bundling implementation as well.